spectre-meltdown-checker.sh

```diff
@@ -662,6 +662,10 @@ check_variant2()
 			dd if=/dev/cpu/0/msr of=/dev/null bs=8 count=1 skip=9 2>/dev/null
 			if [ $? -eq 0 ]; then
 				pstatus green YES
+			elif grep -qi spec_ctrl /proc/cpuinfo; then
+				pstatus green YES
+			elif dmesg | grep -qi "Disabling indirect branch predictor support"; then
```

Comment:
Does this message only appear if and only if the SPEC_CTRL MSR is available? This would mean that if the microcode is not updated, this message doesn't show up at all?